### ISSUE TITLE
PHP 8: Ensure Integration tests run on our GHA libc builds

### ISSUE THREAD

**Issue by Fahmy-Mohammed** [newrelic/newrelic-php-agent#114]
For PHP 8, we've moved our libc builds from Jenkins to Github Actions. Our integration tests that run on Jenkins use some license keys that we pull in from an internal repo. Since this is not possible on Github Actions, we need to find another way to do this. This will involve:

- Investigate and implement another way to pull in the necessary license keys needed to run the integration tests (ideally this can be done using GitHub secrets?)
- Ensure the integration tests run on GHA

Blockers:
- #107

**Comment by zsistla** [comment]
This ticket is part of issue #34.

**Comment by joshuabenuck** [comment]
This ticket ended up being resolved differently than anticipated. Running the tests on GHA results in us being unable to run any of our integration tests for PRs from forks which is an untenable solution.

We ended up updating our CentOS 5 build nodes to build PHP 8.0 as it was the most expedient way to run the integration tests on a system at least as old as the oldest we support (that being CentOS 6).

Once we had PHP 8.0 running, we discovered libc incompatibilities on both our official CentOS 5 build node and an experimental CentOS 6 docker image.

This led to the decision to fallback and do what we have done in previous releases which is to build and test our libc based builds on CentOS 5 (despite it not being an OS we officially support any longer).

To move away from our CentOS 5 infrastructure will require that we create CentOS 6 replacements for those build nodes in addition removing / replacing other CentOS 5 based test systems in use (xPlatform Jenkins jobs perhaps?). I plan on filing a ticket with more details about this at some point.

At this point, both the PR and build jobs have been updated to using the updated CentOS 5 64-bit build node, the makefile targets have been updated to build for PHP 8.0 on the appropriate platforms, and the install script has been updated to check for PHP 8.0 files / detect unsupported configs.

See #132 for the public github related changes. Other changes are in the internal nrcamp / nrlamp repos and the Jenkins job configs.
